packages/SwingSet/test/vat-admin/test-create-vat.js

```diff
@@ -85,46 +85,81 @@ async function doTestSetup(t, enableSlog = false) {
     // for debugging, set enableSlog=true to start tracing after setup
     doSlog = true;
   }
+  if (doVatAdminRestart) {
+    await restartVatAdminVat(c);
+  }
   return { c, id44, idRC, vat13Bundle: bundles.vat13Bundle, hostStorage };
 }
 
-test('createVatByBundle', async t => {
-  const { c, vat13Bundle } = await doTestSetup(t);
+async function testCreateVatByBundle(t, doVatAdminRestart) {
+  const { c, vat13Bundle } = await doTestSetup(t, doVatAdminRestart);
   const kpid = c.queueToVatRoot('bootstrap', 'byBundle', [vat13Bundle]);
   await c.run();
   t.deepEqual(JSON.parse(c.kpResolution(kpid).body), 13);
+}
+
+test('createVatByBundle', async t => {
```

Comment:
with 8 new test cases in this  file, which all launch kernels and a batch of vat workers, I think we might need the same `test.serial` as we use in test-terminate

Reply:
I would say let's not assume there's a problem until there's a problem.  But if we see this exhibiting test flakage in CI we'll immediately know what the first move to make is.